CONTRIBUTING.md

```diff
@@ -103,6 +103,12 @@ for review. Use the following simple PR templates to structure the PR:
 
 * [PR Template](https://github.com/asyml/forte/blob/master/.github/PULL_REQUEST_TEMPLATE.md)
 
+Here is a detailed explanation of each section in the template:
+* The first line should always start with `This PR fixes [issue link]` where `[issue link]` can be replaced with a `#ISSUE_ID` associated to a specific [issue](https://github.com/asyml/forte/issues). This allows Github to automatically link your pull request to the corresponding issue.
```

Comment:
If this PR will close the issue we use "fixes" otherwise we can say "partially addresses"